### DISCUSSION TITLE
Complete Race Data?

### DISCUSSION THREAD

**Discussion by YotaYonder** [br-g/openf1#86] [General]
Hi folks, I'm new to API's and have a question regarding pulling a full race data using JSON. I'm learning Unreal Engine and I wanted to create a leaderboard, racetrack, weather info-graphic, etc and have run through the data from start of the race to the finish. Is my get just to include the date and time from start to finish? Is there an easy way to see the first and last time data point to set that window? Thank you again and I hope to share my unreal engine project with you all when it's in a good spot. Thank you in advance.

**Comment by adityakotha03** [comment]
This is a workaround but you could use the timestamp from lap times call to get starting and ending lap. Again this doesn't include 1st lap but it kind of works.

**Comment by sfollo** [comment]
Hi, I have to develope a webApp with ReactJS for a school project and I'd like to use this API, I have a question. What type of request do I have to call to have the data if the season is over (how can I have the json with the data from the lasts seasons)?
thank you
